### DISCUSSION TITLE
Private Nuget Feeds / Proget Integration

### DISCUSSION THREAD

**Discussion by georgearnall** [badges/shields#6903] [Q&A] [answered]
Hi is there a way to specify custom Nuget feeds? 

We host private Nuget feeds using [Proget](https://inedo.com/proget) but there doesn't seem to be any way to connect to it when self hosting. 

Any ideas?

**Comment by chris48s** [comment, marked as answer]
We don't have this, but there is an open feature request for it: https://github.com/badges/shields/issues/2605
We'd review a PR for it if you're interested in contributing

**Reply by georgearnall** [reply]
Thanks for linking this, I had a quick look but couldn't find it.

I haven't explored the Proget API so not quite sure how much would be involved to get this running. 

In terms of producing a PR, what do you think would be required for an MVP ? 

I'm only interested in Nuget at this time  (and probably only need one badge), although Docker and NPM could be useful down the line. Could be a huge chunk of work if all of this is required.

**Comment by chris48s** [comment]
Starting with only one proget/nuget badge would be ideal tbh. A big PR adding a whole suite of badges is really hard to review, whereas a small PR adding one or two is much easier to read and its more likely to get shipped quickly. More badges can always be added later if someone is motivated to add them.
If you're interested in working on it the tutorial is at https://github.com/badges/shields/blob/master/doc/TUTORIAL.md
